### PR TITLE
Rename CONFIG_TASH_TELNET_INTERFACE to CONFIG_TASH_COMMAND_INTERFACE

### PR DESCRIPTION
**Pull request by jeongchanKim**

### CODE REVIEW

File: apps/shell/Kconfig [hunk 1]
```diff
@@ -18,7 +18,7 @@ config DEBUG_TASH
 	---help---
 		Enable debugging message in TASH.
 
-config TASH_TELNET_INTERFACE
+config TASH_COMMAND_INTERFACE
 	bool "enable telnet interface to TASH"
```
Comment (sunghan-chang) [inline apps/shell/Kconfig]: need to fix here also.

Comment (sunghan-chang) [inline apps/shell/Kconfig]: same at help

File: apps/examples/telnetd/telnetd.c [hunk 2]
```diff
@@ -71,7 +71,7 @@
 /****************************************************************************
  * Pre-processor Definitions
  ****************************************************************************/
-#ifdef CONFIG_TASH_TELNET_INTERFACE
+#ifdef CONFIG_TASH_COMMAND_INTERFACE
 #define MAX_TELNET_CMDS CONFIG_TASH_MAX_COMMANDS
```
Comment (sunghan-chang) [inline apps/examples/telnetd/telnetd.c]: MAX_TELNET_CMDS?

Comment (jeongchanKim) [inline apps/examples/telnetd/telnetd.c]: I think that define can be MAX_TELNET_CMDS

**Pull request merged**